ui.apps/src/main/content/jcr_root/apps/saas-aem-module/clientlibs/wknd-sample/css/styles.less

```diff
@@ -52,61 +67,61 @@
   transition: 0.08s ease-in;
 }
 @media (prefers-color-scheme: dark) {
-  .search .saas-container_results_item {
+  .search .saas-container__results-item {
     background-color: #fff;
   }
 }
 @media only screen and (max-width: 768px) {
-  .search .saas-container_results_item {
+  .search .saas-container__results-item {
     margin: 15px 0 0;
   }
 }
 @media only screen and (min-width: 769px) {
-  .search .saas-container_results_item {
+  .search .saas-container__results-item {
     margin: 0 0 15px 15px;
   }
 }
 @media only screen and (min-width: 992px) {
-  .search .saas-container_results_item:hover {
+  .search .saas-container__results-item:hover {
     -webkit-box-shadow: 0 0 0 1px #ffea00;
     box-shadow: 0 0 0 1px #ffea00;
   }
 }
-.search .saas-container_results_item_description {
+.search .saas-container__results-item-description {
   color: dimgray;
   font-size: 14px;
   line-height: inherit;
   margin-top: 30px;
   text-align: start;
 }
-.search .saas-container_results_item_title {
+.search .saas-container__results-item-title {
   color: #202020;
```

Comment:
maybe we can use variables for the colors, box-shadows, font-sizes, in order to stay consistent

Reply:
@Ivana22 This is just a generated CSS, the aim was to have the saas looking 'good'. We will provide a skeleton later on, meaning it will be up to you, clients to style the way it is needed.